src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
```

Comment:
The constructor should not contain expressions directly. Instead, assign the radius value to a local variable and then pass it to the constructor. This improves readability and maintainability of the code.